### DISCUSSION TITLE
Simultaneous G6/G7 and follow Data

### DISCUSSION THREAD

**Discussion by JD3206** [trupici/G-Watch-Wear#4] [General]
Just as and FYI - if you are running both G6 and G7 at the same time, the follow data received from dexcom picks 1 value (most recent?) when it retrieves the share data. Both of mine are tracking well/together, but I thought it was interesting.

Thanks for keeping this awesome tool going that helps me stay on top of this lifelong adventure!

~Jeff

**Comment by JD3206** [comment]
I initially started with both shares turned off in the apps.  With that, I was getting only G data.

I turned on the G7 share in the G7 app, started getting only G7 data.

Last thing I did was toggle on the G6 share data and then I started receiving both sets of data (3 points in a row thus far).

If you see no further updates - this trend has continued and I'm getting all data after toggling both share options on in both g6/G7 apps. 

The only issue I see with this setup is it tells you the difference between the most G6/G7 data points instead of the last reading which isn't a big as this is my choice to run with both data sets (short term - 20 days total and I'm on day 13).

**Comment by BINAY1963** [comment]
Nice information
